leaf-ui/rappid-extensions/AssignmentsTable.js

```diff
@@ -395,11 +395,7 @@ var LinkRelationshipView = Backbone.View.extend({
 
     render: function () {
         this.$el.html(_.template($(this.template).html())(this.model.toJSON()));
-        // TODO: Write statement to handle this case in script
-        // Or if absTime default changed to null just remove if statement
-        if (this.model.get('absTime') == -1) {
-            this.$('.linkAbsRelation').val('')
-        }
+        this.$('.linkAbsRelation').val('')
```

Comment:
The `if` statement must remain because otherwise save values will not appear in the view. Switch the original if statement to `myNull` instead of `-1`.